reflex/compiler/compiler.py

```diff
@@ -66,7 +67,7 @@ def _compile_theme(theme: dict) -> str:
 
 
 def _is_dev_mode() -> bool:
-    return os.environ.get("REFLEX_ENV_MODE", "dev") == "dev"
+    return not is_prod_mode()
```

Comment:
nit: should we put `_is_dev_mode` in the same file as `is_prod_mode`, and also make their naming consistent with the initial underscore